r/vignettes/arrow.Rmd

```diff
@@ -150,42 +159,53 @@ In the tables, entries with a `-` are not currently implemented.
 | int16             | integer                      |
 | int32             | integer                      |
-| int64             | integer^3^                   |
+| int64             | integer^1^                   |
 | uint8             | integer                      |
 | uint16            | integer                      |
-| uint32            | integer^3^                   |
-| uint64            | integer^3^                   |
-| float16           | -                            |
+| uint32            | integer^1^                   |
+| uint64            | integer^1^                   |
+| float16           | -^2^                         |
 | float32           | double                       |
 | float64           | double                       |
 | utf8              | character                    |
-| binary            | arrow_binary ^5^             |
-| fixed_size_binary | arrow_fixed_size_binary ^5^  |
+| large_utf8        | character                    |
+| binary            | arrow_binary ^3^             |
+| large_binary      | arrow_large_binary ^3^       |
+| fixed_size_binary | arrow_fixed_size_binary ^3^  |
 | date32            | Date                         |
 | date64            | POSIXct                      |
 | time32            | hms::difftime                |
 | time64            | hms::difftime                |
 | timestamp         | POSIXct                      |
-| duration          | -                            |
+| duration          | -^2^                         |
 | decimal           | double                       |
 | dictionary        | factor^4^                    |
-| list              | arrow_list ^6^               |
-| fixed_size_list   | arrow_fixed_size_list ^6^    |
+| list              | arrow_list ^5^               |
+| large_list        | arrow_large_list ^5^         |
+| fixed_size_list   | arrow_fixed_size_list ^5^    |
 | struct            | data.frame                   |
 | null              | vctrs::vctrs_unspecified     |
-| map               | -                            |
-| union             | -                            |
-| large_utf8        | character                    |
-| large_binary      | arrow_large_binary ^5^       |
-| large_list        | arrow_large_list ^6^         |
+| map               | -^2^                         |
+| union             | -^2^                         |
+
+^1^: These integer types may contain values that exceed the range of R's 
+`integer` type (32-bit signed integer). When they do, `uint32` and `uint64` are 
+converted to `double` ("numeric") and `int64` is converted to 
+`bit64::integer64`. This conversion can be disabled (so that `int64` always
+yields a `bit64::integer64` vector) by setting `options(arrow.int64_downcast = FALSE)`.
+
+^2^: Some Arrow data types do not have an R equivalent and will raise an error
+if cast to or mapped to via a schema.  See 
+[this discussion section](#no-compat-type) for an example of this.
```

Comment:
Instead of "See [this discussion section]" could we use something like "See [the XXX section]" (the this to me seems a little bit odd.

Also, where is this supposed to be linking to? I've tried to find the section but haven't yet

Reply:
Fixed now.